decidim-debates/app/controllers/decidim/debates/versions_controller.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Debates
+    # Exposes Proposals versions so users can see how a Proposal/CollaborativeDraft
```

Comment:
```suggestion
    # Exposes Debates versions so users can see how a debate
```

Reply:
@oriolgual you resolved the conversation but did not apply the signature documentation correction :sweat_smile:

Reply:
Oh, I applied the commit from the UI and then I force pushed an removed it 🤦‍♂️